doc/user/content/sql/create-source/json-kinesis.md

```diff
@@ -12,6 +12,9 @@ aliases:
 {{% create-source/intro %}}
 This document details how to connect Materialize to JSON-formatted Kinesis
 streams.
+
+Because this source is still in Beta status, please be sure to test
```

Comment:
Given that the entire product is still in beta, and that only an insane team would release a service using beta infrastructure without testing it, this feels like a slightly different tone than I would take. Maybe something like:
```
{{< warning >}}

Kinesis support is still undergoing active development. If you run into any issues with it, please let us know with a [GitHub issue](https://github.com/MaterializeInc/materialize/issues/new?labels=C-bug&template=bug.md).

{{< /warning >}}
```